superset-frontend/src/explore/App.jsx

```diff
@@ -32,10 +34,12 @@ setupPlugins();
 
 const App = ({ store }) => (
   <Provider store={store}>
-    <div>
-      <ExploreViewContainer />
-      <ToastPresenter />
-    </div>
+    <ThemeProvider theme={supersetTheme}>
+      <div>
```

Comment:
This div doesn't look like it has any business being here, but I don't want to be wrong when it's beside the point of this PR ;)